### ISSUE TITLE
Isolation Question

### ISSUE THREAD

**Issue by mageddo** [salceson/spring-security-example#1]
Have you checked if isolation used at `https://github.com/salceson/spring-security-example/blob/master/src/main/java/com/example/service/UserDetailsServiceImpl.java` have worked for you?

Thanks advance

**Comment by salceson** [comment]
I cannot recall having problems with it. I wrote this code for a friend a little while ago and I don't remember it very well now, but I think that it worked then. Have you encountered some problems with this code?

**Comment by mageddo** [comment]
Thanks for answer, I have not tested your code yet, the point is that I did some tests with a POC and get not ISOLATION to work, I think that can be some Spring Configuration, whatever, for this reason I am looking for a sample that uses ISOLATION and test it. I will test you code and give you a feedback.

If you are interested, [this is my POC](https://github.com/mageddo/spring-boot-mvc-jdbc-template/blob/spring-isolation/src/main/java/com/mageddo/service/CustomerService.java). I've tried with PostgreSQL and HSQLDB
